### ISSUE TITLE
improvevoiceover support

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ifrotz/iosfrotz#220]
```
Hello, I noticed that voiceover doesn't read certain chunks of text, such as 
when a screen doesn't require any imput. could you improve this? thanks.
```

Original issue reported on code.google.com by `sukiletx...@gmail.com` on 16 Jun 2013 at 11:41
